typings/audit.d.ts

```diff
@@ -114,12 +115,9 @@ declare global {
       scoreDisplayMode: ScoringModeValue;
       description: string;
       extendedInfo?: {[p: string]: any};
-      notApplicable?: boolean;
```

Comment:
not sure where the best place to put docstrings for audit authors to see, but maybe add a jsdoc string to `notApplicable` in `Product` to mention it would override `scoreDisplayMode` if provided?

Reply:
yeah done